packages/widget.vite-plugin/src/plugin.ts

```diff
@@ -508,7 +488,7 @@ export function FoundryWidgetVitePlugin(options: Options = {}): Plugin {
               }))
               : [],
             rid: entrypointFileIdToConfigMap[chunk.facadeModuleId].rid,
-            version: packageJsonFile.version ?? "0.0.0",
+            version: widgetVersion,
```

Comment:
This seems to be fine, running build on the examples package does generate the manifest in the dist folder